### ISSUE TITLE
Define what's in/out of scope vs. filtered

### ISSUE THREAD

**Issue by dbrumley** [cloudsecurityalliance/gsd-project#17]
The GSD doesn't define "vulnerability", so it may be unclear to others what is in and out of scope. 

I believe we want to have a superset, and let the user filter based upon what they find useful.  The ability to parse -- thus filter -- is a killer feature IMO.

**Comment by dbrumley** [comment]
Just to put in my $0.02 for the conversation, I think "security finding" would be a good term.  Any "security finding" can be committed, and we have an extended "status" attribute such as whether "accepted", "wont fix", "resolved", "open", "in progress", and so on. Similar to when you file a bug report.

**Comment by athix** [comment]
Agreed. Having as much data as possible, but all clearly labeled and filterable, is the intent of the project from my perspective; providing a centralized open-source place to discover, process/parse, and discuss security findings.

**Comment by JonZeolla** [comment]
I use "finding" in this same way at my company, fairly successfully.  I agree that having structured context which is easily parsable is important, allowing recipients of the information to interpret it however best fits their needs.
